bitnami/zookeeper/README.md

```diff
@@ -251,7 +251,32 @@ Bitnami will release a new chart updating its containers if a new version of the
 
 ### Configure log level
 
-You can configure the ZooKeeper log level using the `ZOO_LOG_LEVEL` environment variable. By default, it is set to `ERROR` because of each readiness probe produce an `INFO` message on connection and a `WARN` message on disconnection.
+You can configure the ZooKeeper log level using the `ZOO_LOG_LEVEL` environment variable or the parameter `logLevel`. By default, it is set to `ERROR` because each use of the liveness probe and the readiness probe produces an `INFO` message on connection and a `WARN` message on disconnection, generating a high volume of noise in your logs.
+
+In order to remove that log noise so levels can be set to ‘INFO’, two changes must be made.
+
+First, ensure that you are not getting metrics via the deprecated pattern of polling 'mntr' on the ZooKeeper client port. The preferred method of polling for Apache ZooKeeper metrics is the ZooKeeper metrics server. This is supported in this chart when setting `metrics.enabled` to `true`.
+
+Second, to avoid the connection/disconnection messages from the probes, you can set custom values for these checks which directly them to the ZooKeeper Admin Server instead of the client port. An example is given below which preserves default probe behavior of this chart. By default, an Admin Server will be started that listens on localhost at 8080 and if you have specified either of `zookeeper.admin.serverAddress` or `zookeeper.admin.serverPort` then change the values of the probe to match. The Admin Server is on by default and you must not have disabled it (via `zookeeper.admin.enableServer`) if you wish to use it for the probes.
+
+```
+customLivenessProbe:
+  exec:
+    command: ['/bin/bash', '-c', 'curl -s -m 2 http://localhost:8080/commands/ruok | grep ruok']
```

Comment:
Why are we grepping different things in the probes?

Reply:
Here's the expected output when the server can host client traffic:
```
$ curl -s http://localhost:8080/commands/ruok
{
  "command" : "ruok",
  "error" : null
}
```
And the output when the server cannot host client traffic but the jvm is still running:
```
$ curl -s http://localhost:8080/commands/ruok
{
  "command" : "ruok",
  "error" : "This ZooKeeper instance is not currently serving requests"
}
```

For liveness, I want to make sure the admin server is alive and responding coherently. That will speak to the overall health of the JVM and does not change its return value when the server/ensemble is undergoing leader election.

For readiness, I want to ensure that the client port is accepting new requests. So we grep for the correct message - namely "null" errors.

Reply:
Thanks for the explanation!